src/components/Button2/index.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
```

Comment:
What is this Button2 component? Is it a replacement for the old Button? If so then I think it'd be best to remove the old Button and call this `Button` as `Button2` can be very confusing